sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/FeatureLayerInputManager.cs

```diff
@@ -0,0 +1,200 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class FeatureLayerInputManager : MonoBehaviour
+{
+    public FeatureData FeatureData;
+
+    [SerializeField] private Transform container;
```

Comment:
Reorganize these so serialized props are above non-serialized props + alphabetical (which you did)